pyvista/utilities/arrays.py

```diff
@@ -33,7 +33,7 @@ def _coerce_pointslike_arg(
         points = np.asarray(points)
 
     if not isinstance(points, np.ndarray):
-        raise TypeError("Given points must be a sequence or an array.")
+        raise TypeError("Given points must be a sequence.")
```

Comment:
Now this is one point where I'd definitely refer to a "numerical array-like" or something similar. Or spell it out as "Given points must be convertible to a numerical array".

Reply:
```suggestion
        raise TypeError("Given points must be convertible to a numerical array.")
```